<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetFilesMaker</class>
 <widget class="QWidget" name="SetFilesMaker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SetFilesMaker</string>
  </property>
  <widget class="QPushButton" name="FileFound">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>135</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="OpenDirect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>125</y>
     <width>135</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Directory</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="TextBrowser">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>400</width>
     <height>140</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="Shadow">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>45</y>
     <width>135</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"> QGroupBox {
     border: 2px groove #A3A3A3;
     border-radius: 3px;
	 margin-top: 2ex; /* оставляем пространство вверху для заголовка */
 }</string>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
